src/overlays/actors/ovl_En_Ru1/z_en_ru1.c

```diff
@@ -32,6 +32,9 @@
 
 #define FLAGS (ACTOR_FLAG_ATTENTION_ENABLED | ACTOR_FLAG_UPDATE_CULLING_DISABLED | ACTOR_FLAG_CAN_PRESS_SWITCHES)
 
+#define ENRU1_SWITCH_FLAG(thisx) PARAMS_GET_U(thisx->actor.params, 8, 8)
+#define ENRU1_TYPE(thisx) PARAMS_GET_U(thisx->actor.params, 0, 8)
```

Comment:
```suggestion
#define ENRU1_SWITCH_FLAG(thisx) PARAMS_GET_U((thisx)->params, 8, 8)
#define ENRU1_TYPE(thisx) PARAMS_GET_U((thisx)->params, 0, 8)
```
and if needed pass `&this->actor` to the macro

see `ENDOOR_GET_TYPE` for example